contracts/PolicyDescriptor.sol

```diff
@@ -19,8 +19,9 @@ contract PolicyDescriptor is IPolicyDescriptor {
      * @return description The URI of the ERC721-compliant metadata.
      */
     function tokenURI(IPolicyManager policyManager, uint256 policyID) external view override returns (string memory description) {
-        address product = policyManager.getPolicyProduct(policyID);
-        string memory productName = IProduct(product).name();
+        //address product = policyManager.getPolicyProduct(policyID);
```

Comment:
If we remove the "product name" how do we distinguish products?

Reply:
on chain it doesnt matter. off chain we'll use some sort of mapping